sycl/doc/extensions/Matrix/dpcpp-joint-matrix.asciidoc

```diff
@@ -219,66 +220,309 @@ q.parallel_for(nd_range<2>(G, L), [=](nd_item<2> item)
    const auto sg_startx = global_idx - item.get_local_id(0);
    const auto sg_starty = global_idy - item.get_local_id(1);
    sub_group sg = item.get_sub_group();
-   joint_matrix<sub_group, int8_t, tM, tK> tA(sg);
+   joint_matrix<int8_t, tM, tK> tA(sg);
    // For B, since current implementation does not support non packed layout,
-   // users need to specify the updated VNNI sizes along with the packed_b layout
-   joint_matrix<sub_group, int8_t, tK, tN, packed_b> tB(sg);
-   joint_matrix<sub_group, int32_t, tM, tN> tC(sg);
-   joint_matrix_load(sg, tC, memC + sg_startx * tM * N + sg_starty, N, matrix_layout::row_major);
+   // users need to specify the packed_b layout
+   joint_matrix<int8_t, tK, tN, packed_b> tB(sg);
+   joint_matrix<int32_t, tM, tN> tC(sg);
+   joint_matrix_load(sg, tC, memC + sg_startx * tM * N + sg_starty/SG_SIZE*tN, N, matrix_layout::row_major);
    for (int k = 0; k < K; k += tk) {
      joint_matrix_load(sg, tA, memA + sg_startx * tM * K + k, K, matrix_layout::row_major);
-     joint_matrix_load(sg, tB, memB + k * N + sg_starty, N, matrix_layout::packed_b); // VNNI
+     joint_matrix_load(sg, tB, memB + k * N + sg_starty/SG_SIZE*tN*4, N*4, matrix_layout::packed_b); // VNNI
      tC = joint_matrix_mad(sg, tA, tB, tC);
    }
-   joint_matrix_store(sg, tC, memC + sg_startx * tM * N + sg_starty, N, matrix_layout::row_major);
+   joint_matrix_store(sg, tC, memC + sg_startx * tM * N + sg_starty/SG_SIZE*tN, N, matrix_layout::row_major);
 }).wait();
-  
 ```
-## Implementation Status
-For oneAPI release 3, an AOT implementation is available on the CPU device to targets AMX hardware. we are using AMX tile intrinsics to implement the matrix load and store operations. Since we are currently emitting AMX intrinsics directly, this only enables AOT compilation. 
 
-Currently, this is the compilation command line needed to invoke AMX unit of Sapphire Rapids CPU:
+== Query Interface
+AMX, DPAS and Nvidia TPUs support different sizes and types. 
+The query interface is used to validate user code and inform them about supported types, sizes, scope, and layouts by the implementation.
+This also offers development and tuning productivity by both scientists and library developers. The query interface we are proposing here is a compile-time query, 
+so there will be no runtime errors.   
+The query interface proposed here consists of three functionalities:
+
+- At compile time, inform the user whether a specific combination is valid or not.
+
+- Construct the matrices using a default shape if user does not provide a combination 
```

Comment:
For people that aren't aware perhaps clarify that this corresponds to the case where the user provides the sizes of large "tile" matrices but doesn't specify the sizes of corresponding submatrices of the "tiles" - Construct the matrices refers to constructing the submatrices of the matrices whose size the user provided.